src/components/organisms/AssetActions/Compute/index.tsx

```diff
@@ -288,6 +288,12 @@ export default function Compute({
         algorithmAssetOrderId,
         selectedAlgorithmAsset.dataToken
       )
+
+      if (!respone) {
```

Comment:
an error will have a `response`. In my case a `500` was returned so check here would be along the lines of:

```js
if (!reponse || !response.ok || response.status !== '200') {
```

Reply:
I'm not sure we can have it exactly as above because response of type ComputeJob, and for eg the succes status is 10, we can also check the statusText to be "Job started"